src/useData.ts

```diff
@@ -46,16 +41,13 @@ function createReturnedTuple<G> (nextDataState: G, updateFn: (cb: UpdateCb<G>) =
 type Data = (Record<string, unknown>|Record<string, unknown>[])
 
 export function useData<T extends Data = Record<string, unknown>> (data: T) {
-  const [nextDataState, setDataState] = useState<T>(data)
+  const [state, setState] = useState(data)
 
-  useEffect(() => {
-    setDataState(data)
-  }, [data])
+  useEffect(() => { setState(data) }, [data])
```

Comment:
Should we actually update the data whenever it changes or should we leave this to the consumer?